config/config.example.yml

```
@@ -0,0 +1,8 @@
+ontologies:
+  - MONDO
+
+columns_to_annotate:
+  - condition_source_text
+
+# Optional output directory
+output_dir: data/output/
```

```suggestion
output_dir: output/
```
Let's just keep everything in the output directory that Patrick set up with `make`. As I mentioned, this is safe from accidental commits to git, too.